.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
+cache: pip
 python:
   - "2.6"
   - "2.7"
 
 before_script:
-  - virtualenv ../venv
```

Comment:
Travis build environment is basically a clean virtual environment already, so this is just an extra step.